docs/app/Examples/collections/Table/States/TableWarningShorthand.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Table } from 'stardust'
+
+const headerRow = [
+  'Name',
```

Comment:
The reason the body row renderer needs to be a function is that each row in the body is a function of an element in the `tableData` array. The header and footer rows are static as far as the `Table` is concerned so I'm not sure if it would make sense for them to be functions.

> I think each row renderer should get access to the row element.

Also to clarify, `headerRow` _is_ the row element, same with `footerRow`. `bodyRow` is a more of a "template" of what a row element looks like.